.NET/Microsoft.Recognizers.Text.DateTime/Utilities/TimexUtility.cs

```diff
@@ -207,5 +207,42 @@ public static DateObject OffsetDateObject(DateObject date, int offset, DatePerio
 
             return ret;
         }
+
+        public static TimeOfDayResolutionResult ParseTimeOfDay(string timex)
```

Comment:
Rename the parameter to tod. It can remain a string, or be an enum.
timex should be a property of the response. It's fine if it's the same string value as passed in. But it may not always be the case.